### PR TITLE
do not submit an empty registry

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

**Comment by sckott** [comment]
@maelle  should I have a look at this PR?

**Comment by maelle** [comment]
Yes please

File: inst/automation/make_registry.sh [hunk 1]
```diff
@@ -19,10 +19,16 @@ if [ $? -eq 0 ]; then
...
-  git commit -am 'registry.json and raw_cm.json updated'
-  git push
+  pkgslength=$(jq .packages ../registry.json | jq length)
+  if [  $pkgslength -gt 250 ]; then
+    # upload new registry files to github
+    echo "pushing changes to github"
+    git commit -am 'registry.json and raw_cm.json updated'
+    git push
+  else
+    echo "Only $pkglength package(s), this is not good. :-("
```
Comment (sckott) [inline inst/automation/make_registry.sh:29]: `pkglength` !=  `pkgslength `

**Review by sckott — CHANGES_REQUESTED** [review]
tested the shell stuff locally - should work - just need to fix that variable mis-spelled

File: inst/automation/make_registry.sh (hunk 1 above)
Comment (maelle) [reply inst/automation/make_registry.sh]: thanks, now fixed.

**Review by sckott — APPROVED** [review]

**Comment by maelle** [comment]
Can you merge or is it dangerous on a Friday? 🙂

**Comment by sckott** [comment]
lets try it, i can roll it back if it doesn't work

**Comment by sckott** [comment]
@maelle everything went smoothly i think 

https://github.com/ropensci-org/makeregistry/runs/1840791741?check_suite_focus=true
https://github.com/ropensci/roregistry/commit/805d8e4c370d7c1bf83d3e2427be575bb03a9512

**Pull request merged**